src/libraries/Microsoft.PowerFx.Connectors/OpenApiExtensions.cs

```diff
@@ -742,21 +773,23 @@ internal static ConnectorDynamicValue GetDynamicValue(this IOpenApiExtensible pa
             // https://learn.microsoft.com/en-us/connectors/custom-connectors/openapi-extensions#use-dynamic-values
             if (param?.Extensions != null && param.Extensions.TryGetValue(XMsDynamicValues, out IOpenApiExtension ext) && ext is OpenApiObject apiObj)
             {
-                ConnectorDynamicValue cdv = new ();
-
-                // Mandatory openrationId for connectors, except when capibility or builtInOperation are defined
-                apiObj.WhenPresent("operationId", (opId) => cdv.OperationId = OpenApiHelperFunctions.NormalizeOperationId(opId));
-                apiObj.WhenPresent("parameters", (opPrms) => cdv.ParameterMap = GetOpenApiObject(opPrms, true));
-                apiObj.WhenPresent("value-title", (opValTitle) => cdv.ValueTitle = opValTitle);
-                apiObj.WhenPresent("value-path", (opValPath) => cdv.ValuePath = opValPath);
-                apiObj.WhenPresent("value-collection", (opValCollection) => cdv.ValueCollection = opValCollection);
-
-                // we don't support BuiltInOperations or capabilities right now
-                // return null to indicate that the call to get suggestions is not needed for this parameter
-                apiObj.WhenPresent("capability", (string op_capStr) => cdv = null);
-                apiObj.WhenPresent("builtInOperation", (string op_bioStr) => cdv = null);
-
-                return cdv;
+                if (apiObj.TryGetValue("parameters", out IOpenApiAny op_prms) && op_prms is OpenApiObject opPrms)

```

Comment:
>if (apiObj.TryGetValue("parameters", out IOpenApiAny op_prms) && op_prms is OpenApiObject opPrms) [](http://example.com/codeflow?start=16&length=97)

It's not a real code change, it's the same as line 749 (in previous version) and simply follows the same pattern used in other GetDynamicXXX methods (consistency)